assets/web/_bevy_dev/auto_reload.js

```diff
@@ -0,0 +1,72 @@
+// Automatically reload when the server restarts
+// Adopted from <https://github.com/trunk-rs/trunk/pull/231>
+(function () {
+  const baseUrl = window.location.host + '/_bevy_dev/websocket';
+  const websocketUrl = 'ws://' + baseUrl;
+  const httpUrl = 'http://' + baseUrl;
+
+  const pollIntervalMs = 5_000;
+
+  /** @type WebSocket | undefined */
+  let webSocket;
+
+  let isFirstLoad = true;
+  let isConnected = false;
+
+  function onOpen() {
+    if (isFirstLoad) {
+      console.info("Connected to dev websocket");
+    } else {
+      console.info("Reconnected to dev websocket");
+      window.location.reload()
+    }
+
+    isFirstLoad = false;
+    isConnected = true;
+  }
+
+  function onClose() {
+    if (isConnected) {
+      console.warn("Lost connection to the dev server");
+    }
+
+    isConnected = false;
+
+    // Periodically try to reconnect to the server
+    window.setTimeout(recreateWebsocket, pollIntervalMs);
```

Comment:
This won't be cleared when the websocket re-establishes connection, but instead when the page is reloaded. This should be fine, but I wanted to bring it up just in case!